.github/workflows/build.yml

```
@@ -0,0 +1,51 @@
+name: Build and test
+
+on:
+  pull_request:
+    branches: [ main ]
+
```

If we want to change up the default shell for all of the run actions, that can be configured once at the root.

```suggestion
defaults:
  run:
    shell: bash
```

---

alright, that's cool